libs/perun/components/src/lib/entity-search-select/entity-search-select.component.ts

```diff
@@ -143,6 +143,7 @@ export class EntitySearchSelectComponent<T extends PerunBean>
         this.entitySelected.emit(value);
       }
     }
+    // this.visibleDeselectButton = !this.disableDeselectButton && (this.selectedEntities.length !== 0 || this.entitySelected != null);
     this.visibleDeselectButton = !this.disableDeselectButton && this.selectedEntities.length !== 0;
```

Comment:
I would prefer for the button to be always visible if it is enabled. Maybe it can be in disabled state if nothing is selected.  But removing it completely seems distracting.
I am also strongly considering we should remove the text from the button. It is a simple action and right now it takes way too much space, especially if we have multiple selections on the screen.